src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSCloudPool.cs

```diff
@@ -364,7 +364,8 @@ public PSNetworkConfiguration NetworkConfiguration
                 this.networkConfiguration = value;
             }
         }
-        
+
+        [Obsolete("This will be removed in a future version and replaced with ResizeErrors")]
         public PSResizeError ResizeError
```

Comment:
@matthchr this comment goes for all properties you plan to make obsolete:

You should add the property that is going to replace the obsolete property. That way users can start using that property immediately and can update their scripts before the breaking change occurs instead of having to wait. If you are simply renaming the property, you can do the following:

```cs
[Obsolete("This will be removed in a future version and replaced with ResizeErrors")]
public PSResizeError ResizeError
{
    return this.ResizeErrors;
}

public PSResizeError ResizeErrors
{
    get
    {
        // previous logic from ResizeError
    }
}

```